data/kaushalnitb.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Kaushal Kishor",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a DevOps engineer"
+  "links": [
+  {
+  "name": "Sara Jaoude",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
+  "links": [
```

Comment:
Your file contains duplicate keys, please remove these duplicates as suggested. 
```suggestion
```

**Pro Tip:** To make the requested changes directly, you can use the <kbd>Commit suggestion</kbd> button above.